lib/model.js

```diff
@@ -22,6 +22,9 @@ _.extend(Model.prototype, {
             callback = options;
             options = {};
         }
+        else if(!options) {
```

Comment:
A minor thing, please could you move this to the previous line. There are a few instances like this.